readme.txt

```diff
@@ -178,6 +178,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Fix - Removed RSVP kitchen sink template and functionality. [ET-1162]
```

Comment:
Not sure if `Kitchen sink` is something that we would want to include in the public readme. 

Maybe we can say something like:

```Removed unused RSVP V2 preview templates. ```

Reply:
I'd avoid having `unused`, but that's a good point. We can use "RSVP V2 preview templates" so that's more understandable what we're doing.

Reply:
Thanks @rafsuntaskin and @juanfra !